Hestia/app/src/androidTest/java/com/rugged/application/hestia/UI/dialogs/ChangeCredentialsDialogTest.java

```diff
@@ -34,9 +34,9 @@
     private final String OLD_PASS = "password";
     private final String NEW_USER = "testuser";
     private final String NEW_PASS = "testpass";
-    private final String LOGOUT_TEXT = HomeActivity.logoutText;
+    private final String LOGOUT_TEXT = "Logout";
     private final String PACKAGE_NAME = "com.rugged.application.hestia";
-    private final String CHANGE_TEXT = HomeActivity.changeCredentialsText;
+    private final String CHANGE_TEXT = "Change user/pass";
```

Comment:
That was not possible at first sight without context. Well, it is possible, but very hard I think, will try it tonight.